backend/api/pretix/types.py

```diff
@@ -1,10 +1,19 @@
+from __future__ import annotations
+
 from datetime import datetime
 from enum import Enum
-from typing import List, Optional
+from typing import Dict, List, Optional
 
 import strawberry
 
 from api.pretix.constants import ASSOCIATION_CATEGORY_INTERNAL_NAME
+from pretix.types import Category as CategoryDict
+from pretix.types import Item as ItemDict
+from pretix.types import Option as OptionDict
+from pretix.types import OrderPosition as OrderPositionDict
+from pretix.types import ProductVariation as ProductVariationDict
+from pretix.types import Question as QuestionDict
+from pretix.types import Quota as QuotaDict
```

Comment:
weird it didn't get formatted in one import

Reply:
There is a specific configuration for `as` imports: `combine_as_imports` by default is False!